### PR TITLE
add exer 14

### PR DESCRIPTION
**Pull request by Tayoto-Mikaella**
Please review my file. Thank you

### CODE REVIEW

File: tayoto_mikaellaantonette/script.js [hunk 1]
```diff
@@ -47,3 +47,91 @@ sortAscButton.addEventListener("click", function () {
 sortDescButton.addEventListener("click", function () {
   sortComments(false);
 });
+
+function searchCountry() {
+  var countryName = document.getElementById("countryInput").value.trim();
+  if (!countryName) {
+    document.getElementById("countryDetails").innerHTML =
```
Comment (Villa-VillaDiana) [inline tayoto_mikaellaantonette/script.js]: IDs should be written in snake case. All lower case separated by underscore.

Reference: https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g24da8de92d7_0_5

File: tayoto_mikaellaantonette/script.js [hunk 2]
```diff
@@ -47,3 +47,91 @@ sortAscButton.addEventListener("click", function () {
 sortDescButton.addEventListener("click", function () {
   sortComments(false);
 });
+
+function searchCountry() {
+  var countryName = document.getElementById("countryInput").value.trim();
```
Comment (Villa-VillaDiana) [inline tayoto_mikaellaantonette/script.js]: IDs should be written in snake case. All lower case separated by underscore.

Reference: https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g24da8de92d7_0_5

File: tayoto_mikaellaantonette/script.js [hunk 3]
```diff
@@ -47,3 +47,91 @@ sortAscButton.addEventListener("click", function () {
...
+  if (!countryName) {
+    document.getElementById("countryDetails").innerHTML =
+      "<p>Please enter a country name.</p>";
+    document.getElementById("sameRegionCountries").innerHTML = "";
```
Comment (Villa-VillaDiana) [inline tayoto_mikaellaantonette/script.js]: IDs should be written in snake case. All lower case separated by underscore.

Reference: https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g24da8de92d7_0_5

File: tayoto_mikaellaantonette/script.js [hunk 4]
```diff
@@ -47,3 +47,91 @@ sortAscButton.addEventListener("click", function () {
...
+        country.subregion ? country.subregion : "N/A"
+      }</p>
+
+      <p><strong>Capital:</strong> ${
+        country.capital ? country.capital[0] : "N/A"
+      }</p>
+
+      <p><strong>Timezones:</strong> ${
+        country.timezones ? country.timezones.join(", ") : "N/A"
+      }</p>`;
```
Comment (Villa-VillaDiana) [inline tayoto_mikaellaantonette/script.js]: When splitting the elements into multiple lines, make sure that the continuation line is indented.

Reference: https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g29e6ce31160_0_11

File: tayoto_mikaellaantonette/script.js [hunk 5]
```diff
@@ -47,3 +47,91 @@ sortAscButton.addEventListener("click", function () {
...
+    .then(function (response) {
+      if (!response.ok) {
+        throw new Error("Country not found");
+      }
+      return response.json();
+    })
+
+    .then(function (countryData) {
+      var country = countryData[0];
+      var details = `h2>Country Details - ${country.name.common}</h2>
```
Comment (Villa-VillaDiana) [inline tayoto_mikaellaantonette/script.js]: Always use "let" keyword in declaring variables over the "var" keyword.

Reference: https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g25d183194fb_0_0

File: tayoto_mikaellaantonette/script.js [hunk 6]
```diff
@@ -47,3 +47,91 @@ sortAscButton.addEventListener("click", function () {
...
+      var region = regionData[0].region;
+      var sameRegionCountriesList = regionData
+        .map(function (c) {
+          return `
+          <div class="country-card">
+            <img src="${c.flags.svg}" 
+              alt="Flag of ${c.name.common}" 
+              width="50">
+            <p>${c.name.common}</p>
+          </div>`;
```
Comment (Villa-VillaDiana) [inline tayoto_mikaellaantonette/script.js]: When splitting the elements into multiple lines, make sure that the continuation line is indented.

Reference: https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g29e6ce31160_0_11

File: tayoto_mikaellaantonette/script.js [hunk 7]
```diff
@@ -47,3 +47,91 @@ sortAscButton.addEventListener("click", function () {
...
+        })
+
+        .join("");
```
Comment (Villa-VillaDiana) [inline tayoto_mikaellaantonette/script.js]: I believe you can have them on a single line.

File: tayoto_mikaellaantonette/script.js [hunk 8]
```diff
@@ -47,3 +47,91 @@ sortAscButton.addEventListener("click", function () {
...
+      document.getElementById("countryDetails").innerHTML =
+        "<p>An error occurred: " + error.message + "</p>";
+      document.getElementById("sameRegionCountries").innerHTML = "";
+    });
+}
+
+document.getElementById("countries-button")
+  .addEventListener("click", function () {
+    window.location.href = "./countries.html";
+  });
```
Comment (Villa-VillaDiana) [inline tayoto_mikaellaantonette/script.js:135]: Observe proper indentation

**Review by Villa-VillaDiana — CHANGES_REQUESTED** [review]

File: tayoto_mikaellaantonette/index.html [hunk 9]
```diff
@@ -25,7 +25,11 @@ <h2>My Interests and Hobbies</h2>
             </ul>
           </div>
           <hr />
-          <div class="work">
+          <div class="countries">
+            <button id="countries-button">Countries</button>
+        </div>
```
Comment (Villa-VillaDiana) [inline tayoto_mikaellaantonette/index.html:30]: Observe proper indentation

**Review by Villa-VillaDiana — CHANGES_REQUESTED** [review]

**Review by Villa-VillaDiana — APPROVED** [review]
Looks good to me, you can now merge this

**Review by Villa-VillaDiana — APPROVED** [review]
You can now merge this :))

**Comment by Tayoto-Mikaella** [comment]
Thank you! ^^

**Pull request merged**